python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -48,6 +48,10 @@ def from_pretrained(cls,
         if bigdl_transformers_int4:
             # Avoid KeyError
             kwargs["ignore_mismatched_sizes"] = True
+            # Speed up when loading model
+            kwargs["low_cpu_mem_usage"] = True
```

Comment:
need to refactor these kwargs:
- "low_cpu_mem_usage" needs to be set to true if `load_in_4bit` or `bigdl_transformers_int4` is true
- "subfolder" and "variant" are only needed when `bigdl_transformers_int4` is true

Reply:
fixed